cmake/SpectreSetupPython.cmake

```diff
@@ -237,3 +270,18 @@ function(SPECTRE_ADD_PYTHON_TEST TEST_NAME FILE TAGS)
     ENVIRONMENT "PYTHONPATH=${SPECTRE_PYTHON_PREFIX_PARENT}:\$PYTHONPATH"
     )
 endfunction()
+
+# Register a python test file that uses bindings with ctest.
+# - TEST_NAME    The name of the test,
+#                e.g. "Unit.DataStructures.Python.DataVector"
+#
+# - FILE         The file to add, e.g. Test_DataVector.py
+#
+# - TAGS         A semicolon separated list of labels for the test,
+#                e.g. "unit;DataStructures;python"
```

Comment:
why aren't these names CamelCased?

Reply:
just copied from the function below. Fixed both.